docs/README.md

```diff
@@ -8,3 +8,39 @@ jobs that the bootstrapper does:
 - Run the kubelet as a windows service
 
 This will be remotely invoked from a Ansible script or can be run locally
+
+## Requirements
+
+- Must be run on Windows server 2019
+- Must be run as administrator
+- A worker ignition file generated by the cluster must be on disk
+- The kubelet you wish to use must be on disk
+
+## Usage
+```
+make build
+```
+
+```
+wmcb run --ignition-file $IGNITION_FILE_PATH --kubelet-path $KUBELET_PATH
+```
+
+## Testing
+
+On an existing Windows instance which is ready to join the cluster, copy the worker ignition file to C:\Windows\Temp\worker.ign, and the kubelet to C:\Windows\Temp\kubelet.exe
+
+On the Windows instance, run:
+```
```

Comment:
Please address this in another PR. Let the build be done on any host but skip running tests on Linux nodes